scripts/mixins/families/ghrah.lua

```diff
@@ -3,126 +3,124 @@ require('scripts/globals/mixins')
 
 g_mixins = g_mixins or {}
 
+local skinConfig = {
+    [1161] = { -- Fire
+        spellList = 484,
+        mods = { { xi.mod.ICE_MEVA, 80 }, { xi.mod.PARALYZERES, 99 }, { xi.mod.BINDRES, 99 }, { xi.mod.FIRE_MEVA, 100 }, { xi.mod.WATER_MEVA, -27 } }
+    },
+    [1162] = { -- Ice
+        spellList = 479,
+        mods = { { xi.mod.WIND_MEVA, 80 }, { xi.mod.GRAVITYRES, 99 }, { xi.mod.SILENCERES, 99 }, { xi.mod.ICE_MEVA, 100 }, { xi.mod.PARALYZERES, 100 }, { xi.mod.BINDRES, 100 }, { xi.mod.FIRE_MEVA, -27 } }
+    },
+    [1163] = { -- Wind
+        spellList = 480,
+        mods = { { xi.mod.EARTH_MEVA, 80 }, { xi.mod.SLOWRES, 99 }, { xi.mod.WIND_MEVA, 100 }, { xi.mod.GRAVITYRES, 100 }, { xi.mod.SILENCERES, 100 }, { xi.mod.ICE_MEVA, -27 } }
+    },
+    [1164] = { -- Earth
+        spellList = 481,
+        mods = { { xi.mod.THUNDER_MEVA, 80 }, { xi.mod.STUNRES, 99 }, { xi.mod.EARTH_MEVA, 100 }, { xi.mod.SLOWRES, 100 }, { xi.mod.WIND_MEVA, -27 } }
+    },
+    [1165] = { -- Lightning
+        spellList = 482,
+        mods = { { xi.mod.WATER_MEVA, 80 }, { xi.mod.POISONRES, 99 }, { xi.mod.THUNDER_MEVA, 100 }, { xi.mod.STUNRES, 100 }, { xi.mod.EARTH_MEVA, -27 } }
+    },
+    [1166] = { -- Water
+        spellList = 483,
+        mods = { { xi.mod.FIRE_MEVA, 80 }, { xi.mod.WATER_MEVA, 100 }, { xi.mod.POISONRES, 100 }, { xi.mod.THUNDER_MEVA, -27 } }
+    },
+    [1167] = { -- Light
+        spellList = 478,
+        mods = { { xi.mod.LIGHT_MEVA, 100 }, { xi.mod.LULLABYRES, 100 }, { xi.mod.DARK_MEVA, -27 } }
+    },
+    [1168] = { -- Dark
+        spellList = 477,
+        mods = { { xi.mod.DARK_MEVA, 100 }, { xi.mod.SLEEPRES, 100 }, { xi.mod.LIGHT_MEVA, -27 } }
+    }
+}
+
+local function getTargetForm(mob)
+    local targetForm = 2
+
+    if mob:getZone():getID() == xi.zone.GRAND_PALACE_OF_HUXZOI then
+        targetForm = mob:getXPos() > 0 and 2 or 3
+    elseif mob:getZone():getID() == xi.zone.THE_GARDEN_OF_RUHMET then
+        targetForm = mob:getZPos() < 0 and 2 or 3
+    end
```

Comment:
local zoneId = mob:getZone():getID()

Also, we are already setting targetForm to 2 by default.
So
```lua
local function getTargetForm(mob)
    local targetForm = 2
    local zoneId     = mob:getZone():getID()

    if
        (zoneId == xi.zone.GRAND_PALACE_OF_HUXZOI and mob:getXPos() <= 0) or
        (zoneId == xi.zone.THE_GARDEN_OF_RUHMET and mob:getZPos() >= 0)
    then
        targetForm = 3
    end

    return targetForm
end
```

I'm pretty sure this kind of stuff has been mentioned in the past.